client/src/components/assessments/instant/InstantAssessmentsContainerField.js

```diff
@@ -105,31 +105,58 @@ const InstantAssessmentsContainerField = ({
   formikProps,
   canRead,
   canWrite,
-  readonly
+  readonly,
+  showEntitiesWithoutAssessments
 }) => {
   const { values } = formikProps
+  function sortEntries(e1, e2) {
+    let e1HasAssessments = false
+    const e1entityInstantAssessments = e1.getInstantAssessments()
+    e1entityInstantAssessments.forEach(([ak, ac]) => {
```

Comment:
I would use `return e1entityInstantAssessments.some(…) ? 1 : -1` instead of looping over all of them and setting a boolean variable.